### PR TITLE
[go-borges] fix remaining tests

### LINKED ISSUE #857: [go-borges] Fix tests in integration
* ~~Create branch in gitbase repository with the initial integration from https://github.com/jfontan/gitbase/tree/go-borges-go-rebased~~ is `go-borges-integration`
* Tests from `cmd/gitbase/command/server_test.go` are still not modified to use go-borges. I'm not sure these are needed anymore.
* Tests that use `closedSiva` can not work with go-borges (`testTableIndexIterClosed` and `testTableIterClosed`). They may be moved to go-borges.
* `TestRefsIterSiva` fails and I'm not sure what's needed to make it work.

```
--- FAIL: TestRefsIterSiva (0.00s)
    --- FAIL: TestRefsIterSiva/all_refs (0.00s)
        require.go:97: 
            	Error Trace:	squash_iterator_test.go:989
            	Error:      	element [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] appears more times in [[/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3]] than in [[015da2f4-6d89-7ec8-5ac9-a38329ea875b HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [015da2f4-6d89-7ec8-5ac9-a38329ea875b refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3]]
            	Test:       	TestRefsIterSiva/all_refs
    --- FAIL: TestRefsIterSiva/repo_refs (0.00s)
        require.go:97: 
            	Error Trace:	squash_iterator_test.go:989
            	Error:      	element [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] appears more times in [[/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3]] than in [[015da2f4-6d89-7ec8-5ac9-a38329ea875b HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [015da2f4-6d89-7ec8-5ac9-a38329ea875b refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3]]
            	Test:       	TestRefsIterSiva/repo_refs
    --- FAIL: TestRefsIterSiva/remote_refs (0.00s)
        require.go:97: 
            	Error Trace:	squash_iterator_test.go:1013
            	Error:      	element [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] appears more times in [[/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3] [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3] [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [/home/jfontan/work/gitbase/_testdata/05893125684f2d3943cd84a7ab2b75e53668fba1.siva refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3]] than in [[015da2f4-6d89-7ec8-5ac9-a38329ea875b HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [015da2f4-6d89-7ec8-5ac9-a38329ea875b refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3] [015da2f4-6d89-7ec8-5ac9-a38329ea875b HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [015da2f4-6d89-7ec8-5ac9-a38329ea875b refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3] [015da2f4-6d89-7ec8-5ac9-a38329ea875b HEAD dbfab055c70379219cbcf422f05316fdf4e1aed3] [015da2f4-6d89-7ec8-5ac9-a38329ea875b refs/heads/master dbfab055c70379219cbcf422f05316fdf4e1aed3]]
            	Test:       	TestRefsIterSiva/remote_refs
```

### PR DESCRIPTION
**Pull request by jfontan**
* `commit_stats_test`
* `fs_error_test`

Fixes #857 

 - [x] I updated the documentation explaining the new behavior if any.
 - [x] I updated CHANGELOG.md file adding the new feature or bug fix.
 - [x] I updated go-mysql-server using `make upgrade` command if applicable.
 - [x] I added or updated examples if applicable.
 - [x] I checked that changes on schema are reflected into the documentation, if applicable.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #================================
 # Stage 1: Build Gitbase
 #================================
-FROM golang:1.12-alpine as builder
+FROM golang:1.11-alpine as builder
```
Comment (ajnavarro) [inline Dockerfile]: this is wrong, isn't it?

File: .travis.yml [hunk 2]
```diff
@@ -2,6 +2,9 @@ language: go
 
 go_import_path: github.com/src-d/gitbase
 
+env:
```
Comment (ajnavarro) [inline .travis.yml]: This might come from master instead as new line from this PR

File: Dockerfile (hunk 1 above)
Comment (jfontan) [reply Dockerfile]: I've copied the `Dockerfile` version from master. It uses 1.11.

File: .travis.yml (hunk 2 above)
Comment (jfontan) [reply .travis.yml]: Yes, this file is copied from master so tests doesn't complain.

**Pull request merged**